staging/publishing/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
+- sttts
+- nikhita
+- dims
+reviewers:
+- sttts
+- dims
+- nikhita
+labels:
+- sig/release
+- area/release-infra
```

Comment:
one more small change is needed - this label is now called `area/release-eng`. :grimacing:

Reply:
fixed